CMSgov/design-system
# No ticket: Use popperjs Placement type for Tooltip

Replace manually-added tooltip placement options with the TS type definition from @popperjs/core

**Note:** 
- Add the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) to the PR title like this `[WNMGDS-10] - title of pr here` to link to the related issue in Jira.
- You can automatically [close related GitHub issues by using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- A demo doc site is automatically generated for this branch at `http://design-system-demo.s3-website-us-east-1.amazonaws.com/{branch_name}`. For example, the demo for `WNMGDS-10/feature` will be located at `http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-10/feature`.
- If your changes involve code please update the snapshots by running `yarn update-snapshots`.


**Please follow the format below and remove any sections that aren't relevant.**
## Summary
Updates the `placement` prop for the `Tooltip` component. Before conversion to TS, this prop took one of a subset of popper's `Placement` type values. Now that the TS conversion is done, this prop can take any value of type `Placement` from `@popperjs/core`. MCT is already using the Tooltip with a non-standard value of `'top-start'`, which throws console errors (in development). This update will eliminate those errors.

## How to test
Instructions on how to test the changes. This is not exhaustive list of ways you should test this PR.

In any child project, or a standalone React/TS test app, render a `Tooltip` with a `placement` value from popper, like `top-start`. There should be no TS errors and the placement should work as intended. See https://github.cms.gov/CMS-MCT/coverage-tools-frontend#testing-design-system-updates for instructions on testing locally with MCT, however, unsure if these will currently work with recent updates to the MCT repo.
